tests/paramgrill.c

```diff
@@ -27,7 +27,8 @@
 #include "xxhash.h"
 #include "util.h"
 #include "bench.h"
-
+#include "zstd_errors.h"
+#include "zstd_internal.h"
 
 /*-************************************
```

Comment:
`paramgrill.c` rights seem to have unduly changed from `644` to `755` (executable).
Should be fixed, to remain `644`.